lecture2gether-vue/src/components/VideoMetaData.vue

```diff
@@ -1,15 +1,28 @@
 <template>
-    <div class="video-meta-data" style="visibility: hidden;">
-        <div class="title">
+    <div class="video-meta-data pa-6">
+        <div class="video-title display-1"
+             :class="{ hidden: videoMetaData.title === null }"
+        >
```

Comment:
Maybe this could be a link to `videoMetaData.url`

Reply:
or we could have a small button next to it with the [open-in-new](http://materialdesignicons.com/icon/open-in-new) icon

Reply:
Added a button with the [open-in-new](http://materialdesignicons.com/icon/open-in-new) icon